### PR TITLE
Add note for CORS: Wildcard Headers and Methods feature support

### PR DESCRIPTION
**Pull request by ethanent**
Many browsers do not support wildcard "Access-Control-Allow-Headers" and "Access-Control-Allow-Methods".

### CODE REVIEW

File: features-json/cors.json [hunk 1]
```diff
@@ -125,14 +125,14 @@
...
       "60":"y",
-      "61":"y #4",
-      "62":"y #4",
-      "63":"y #4",
-      "64":"y #4",
-      "65":"y #4",
-      "66":"y #4",
-      "67":"y #4",
-      "68":"y #4",
+      "61":"y #4 #5",
```
Comment (kiding) [inline features-json/cors.json]: This is a bit weird to me. Looks like 4 and 5 suddenly appeared at version 61.

Comment (kiding) [inline features-json/cors.json]: This is a bit weird to me. Looks like 4 and 5 suddenly appeared at version 61.

The same goes to the other changes; 5 suddenly appeared at version X, then disappeared at version Y.

**Comment by Elchi3** [comment]
fwiw, this is also in caniuse via the MDN data, see https://caniuse.com/#search=wildcard

**Comment by ethanent** [comment]
@Elchi3 That is interesting. It's a bit unfortunate though because one wouldn't know to look for it currently when looking for CORS support in browsers.

**Pull request closed without merging**